src/cli.ts

```diff
@@ -1,160 +1,243 @@
     }, (args: any) => {
-        return new CreatePackageCommand().run(args);
+        return new ExecCommand(
+            'stage|zip|delete|close|sideload',
+            args
+        ).run();
     })
 
-    .command('publish', 'Publish a pre-existing packaged zip file to a remote Roku', (builder) => {
+    .command('package', 'execute build actions for packaging app', (builder) => {
         return builder
+            .option('rootDir', { type: 'string', description: 'The selected root folder to be copied', demandOption: false })
+            .option('outDir', { type: 'number', description: 'The output directory', demandOption: false })
             .option('host', { type: 'string', description: 'The IP Address of the host Roku', demandOption: false })
             .option('password', { type: 'string', description: 'The password of the host Roku', demandOption: false })
+            .option('host', { type: 'string', description: 'The IP Address of the host Roku', demandOption: false })
+            .option('remoteport', { type: 'number', description: 'The port to use for remote', demandOption: false })
+            .option('timeout', { type: 'number', description: 'The timeout for the command', demandOption: false })
+            .option('remoteDebug', { type: 'boolean', description: 'Should the command be run in remote debug mode', demandOption: false })
+            .option('remoteDebugConnectEarly', { type: 'boolean', description: 'Should the command connect to the debugger early', demandOption: false })
+            .option('failOnCompileError', { type: 'boolean', description: 'Should the command fail if there is a compile error', demandOption: false })
+            .option('retainDeploymentArchive', { type: 'boolean', description: 'Should the deployment archive be retained', demandOption: false })
             .option('outDir', { type: 'string', description: 'The output directory', demandOption: false })
-            .option('outFile', { type: 'string', description: 'The output file', demandOption: false });
+            .option('outFile', { type: 'string', description: 'The output file', demandOption: false })
+            .option('deleteDevChannel', { type: 'boolean', description: 'Should the dev channel be deleted', demandOption: false })
+            .option('signingPassword', { type: 'string', description: 'The password of the signing key', demandOption: false })
+            .option('stagingDir', { type: 'string', description: 'The selected staging folder', demandOption: false });
     }, (args: any) => {
-        return new PublishCommand().run(args);
+        return new ExecCommand(
+            'close|rekey|stage|zip|delete|close|sideload|squash|sign',
+            args
+        ).run();
+    })
+
+    .command('exec', 'larger command for handling a series of smaller commands', () => {
+        return attachCommonArgs;
```

Comment:
Let's just go ahead and inline the options here.

Reply:
Done